src/definitions/modules/dropdown.less

```diff
@@ -616,13 +616,22 @@ select.ui.dropdown {
         .ui.ui.selection.dropdown@{notInverted}:has(> input:focus) {
             border-color: @selectionFocusBorderColor;
             box-shadow: none;
-            & > i.icon {
+            & > i.icon:not(.remove) {
                 opacity: @selectionIconFocusOpacity;
             }
         }
-    }
-    .ui.ui.selection.dropdown:focus > i.icon {
-        opacity: @selectionIconFocusOpacity;
+
+        .ui.ui.selection.dropdown {
+            &:focus,
+            &:has(> input:focus) {
+                & > i.icon:not(.remove) {
+                    opacity: @selectionIconFocusOpacity;
+                }
+                & > i.icon.remove:not(:hover) {
+                    opacity: calc(@actionIconFocusOpacity);
```

Comment:
The calc method should be either moved to the variable declaration itself or escaped here if your intention is to generate "calc()" as real CSS property.
calc() itself is a LESS function, so in some situations the value is calculated by LESS itself. Don't know your intention, but to allow the user to define anything as opacity we should move the calc() inside the variable.

Here is a short overview about the differences
![image](https://github.com/user-attachments/assets/47723bf2-f7d7-47e6-aa2a-8e19f4fe6248)

Reply:
Already fixed.

Reply:
Sorry. I just remove `calc()` as I think it's unnecessary. We can safely use the evaluated value from LESS.